### PR TITLE
Remove some more MatcherAssert

### PR DESCRIPTION
**Pull request by victornoel**
This is for #1391.

### CODE REVIEW

**Comment by 0crat** [comment]
This pull request [#1393](https://github.com/yegor256/cactoos/issues/1393) is assigned to @marceloamadeu[/z](https://www.0crat.com/u/marceloamadeu), here is [why](https://www.0crat.com/footprint/C63314D6Z/7ceaf76e-b701-4a72-8d6e-e50fc5c18b42); the budget is 15 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§27](http://www.zerocracy.com/policy.html#27) and when you decide to accept the changes, inform @paulodamaso[/z](https://www.0crat.com/u/paulodamaso) (the architect) right in this ticket; if you decide that this PR should not be accepted ever, also inform the architect; this [blog post](http://www.yegor256.com/2015/02/09/serious-code-reviewer.html) will help you understand what is expected from a code reviewer; there will be no monetary reward for this job

File: src/test/java/org/cactoos/scalar/FallbackFromTest.java [hunk 1]
```diff
@@ -26,52 +26,51 @@
...
 public final class FallbackFromTest {
 
     @Test
     public void supportsException() {
-        MatcherAssert.assertThat(
-            "IOException class is not supported, but should be",
+        new Assertion<>(
+            "Must support IOException class",
             new FallbackFrom<>(
                 new IterableOf<>(IOException.class),
```
Comment (marceloamadeu) [inline src/test/java/org/cactoos/scalar/FallbackFromTest.java:46]: @victornoel error log: `unchecked generic array creation for varargs parameter of type java.lang.Class<? extends java.lang.Throwable>[]`

File: src/test/java/org/cactoos/scalar/FallbackFromTest.java [hunk 2]
```diff
@@ -26,52 +26,51 @@
...
     }
 
     @Test
     public void supportsInheritedException() {
-        MatcherAssert.assertThat(
-            "RuntimeException class is not supported, but should be",
+        new Assertion<>(
+            "Must support RuntimeException class",
             new FallbackFrom<>(
                 new IterableOf<>(RuntimeException.class),
```
Comment (marceloamadeu) [inline src/test/java/org/cactoos/scalar/FallbackFromTest.java:58]: @victornoel error log: `unchecked generic array creation for varargs parameter of type java.lang.Class<? extends java.lang.Throwable>[]`

File: src/test/java/org/cactoos/scalar/FallbackFromTest.java [hunk 3]
```diff
@@ -26,52 +26,51 @@
...
     }
 
     @Test
     public void doesNotSupportException() {
-        MatcherAssert.assertThat(
-            "RuntimeException class is supported, but should not be",
+        new Assertion<>(
+            "Must support RuntimeException class",
             new FallbackFrom<>(
                 new IterableOf<>(RuntimeException.class),
```
Comment (marceloamadeu) [inline src/test/java/org/cactoos/scalar/FallbackFromTest.java:70]: @victornoel error log: `unchecked generic array creation for varargs parameter of type java.lang.Class<? extends java.lang.Throwable>[]`

**Comment by victornoel** [comment]
@marceloamadeu it is fixed, also there were some other qulice errors now fixed

**Comment by victornoel** [comment]
@paulodamaso ping

**Comment by paulodamaso** [comment]
@rultor merge

**Comment by 0crat** [comment]
@sereshqua[/z](https://www.0crat.com/u/sereshqua) please review this job completed by @marceloamadeu[/z](https://www.0crat.com/u/marceloamadeu), as in [§30](http://www.zerocracy.com/policy.html#30); the job will be fully closed and all payments will be made when the quality review is completed

**Comment by sereshqua** [comment]
@0crat quality good

**Pull request merged**